pkg/evpn/evpn.go

```diff
@@ -33,12 +34,22 @@ type Server struct {
 
 // NewServer creates initialized instance of EVPN server
 func NewServer() *Server {
+	nLink := utils.NetlinkWrapper{}
+	return NewServerWithArgs(&nLink)
+}
+
+// NewServerWithArgs creates initialized instance of EVPN server
+// with externally created Netlink for mockery
+func NewServerWithArgs(nLink utils.Netlink) *Server {
```

Comment:
Will this function be used outside `pkg/evpn`? If no, then the function can be made unexported. If yes, then I'd prefer to have the description adjusted and "for mockery" part is removed.

Reply:
agree, fixed